char_gen.go

```diff
@@ -93,7 +93,7 @@ var (
 	MaxFailRate = 1.0 / 1000000000 // Maximum acceptable failure rate after MaxTrials
 )
 
-func (r CharRecipe) acceptableFailRate() (bool, float32) {
+func (r CharRecipe) hasAcceptableFailRate() (bool, float32) {
 	sp := r.SuccessProbability()
```

Comment:
Thanks. Definitely a good change. I really need to learn these sorts of naming conventions.